core/blockchain.go

```diff
@@ -181,6 +297,17 @@ func (self *BlockChain) LastBlockHash() common.Hash {
 	return self.currentBlock.Hash()
 }
 
+// CurrentHeader retrieves the current head header of the canonical chain. The
+// header is retrieved from the chain manager's internal cache.
```

Comment:
In this and the two following comments, please rename chain manager to it's shiny new `blockchain` name :)